index.js

```diff
@@ -34,7 +32,7 @@ const takeUntil = sSrc => src => (start, sink) => {
     }
 
     t === 2 && sTalkback(2);
-    !done && sink(t, d);
```

Comment:
i couldnt find any justification for this one either, we can receive anything from the source when we are in "done" state